templates/partials/filters/ie-report-types.html

```diff
@@ -3,11 +3,11 @@
       <legend class="label" for="report_type">Report type</legend>
       <ul>
         <li>
-          <input id="notice-checkbox-1" name="is_notice" type="checkbox" value="true">
-          <label for="notice-checkbox-1">24 and 48 hour notices</label>
+          <input id="notice-checkbox-1" name="is_notice" type="radio" value="true">
```

Comment:
If these are radio buttons, will it be annoying to un-select them after choosing one or the other? Is there a way to clear this filter without scrolling to the top or bottom of the page to find the clear button? Also, this whole filter is commented out for now.